### ISSUE TITLE
Where to download s100 sample data?

### ISSUE THREAD

**Issue by Edassap** [S-100ExpertTeam/OpenS100#12]
All enc data I downloaded from internet are s57 format, please give me a website or sample data for testing

**Comment by gorogara** [comment]
This is sample data for S-101 ENC.

You can also find it at [KHOA S-100 Viewer](https://github.com/S-100ExpertTeam/khoa-s100-viewer)

[101KR004X0000.zip](https://github.com/S-100ExpertTeam/OpenS100/files/11425861/101KR004X0000.zip)

**Comment by FriedGD** [comment]
Hello, I built this program and downloaded the sample data given by @gorogara, but the program consistently fails to load the file.
The same issue occurs even with other test datasets from IHO and other organizations.
Is there any way to resolve this problem?

**Comment by MikusRL** [comment]
Probably the software has not caught up yet with the latest IHO published signed Feature and Portrayal Catalogs, that the latest test datasets are created in. Latest ones are published here: https://registry.iho.int/productspec/list.do Hopefully the tool alows you to import these and then the soft should allow you to import the latest S-101 prod spec ed.2.0.0) datasets as well.

Just as an additional info for test datasets:
This is a good location to start searching for more official test datasets:
https://iho.int/en/resources-0

IHO GitHub is a good resource to know what goes on in the S-100 standard and its product specification development. This is IHO GitHub pages where the development of the test datasets occur, which will be used in S100ECDIS certification later:
https://github.com/iho-ohi/S-164-Sub-Group
At this IHO GitHub of S-101 are also some test dataset developments downloadable:
https://github.com/iho-ohi/S-101-Test-Datasets

These are some other resources available that IHO has been informed about:
https://github.com/iho-ohi/S100Resources

Then you should also register at IHO portal https://portal.iho.int/login and then you will get the notifications like this one:

> The IHO invites you to download S-100 Test Datasets (TDS) to support the implementation of S-100 systems.
> To receive the datasets, please complete and submit the form below. A download link will be sent to your registered email address upon submission.
>     [Request Form](https://forms.office.com/Pages/ResponsePage.aspx?id=dPLbMWNQnkaut8CrpWiqyfCzNeXO7NdLqMWr6qmIbtVUM1NENkdXUU4zTktIUEdPTVlZT0FVWUdFQy4u)
> 
> Best regards,
> Yong BAEK
> 
> Please note that you are receiving this email as a registered member of the IHO Portal who has consented to receive communications from IHO. Should you wish to discontinue receiving emails from the Portal, you may do so by closing or disabling your IHO Portal account at your convenience. We appreciate your continued participation and understanding.
> 

Hope this helps.

**Comment by gorogara** [comment]
@FriedGD 
Hello, thank you for trying the program.
Could you please share which dataset you used and what error message or something appears when the file fails to load?
If I know the dataset and the exact error details, I may be able to help you further.

**Comment by FriedGD** [comment]
@gorogara
Thank you for response.
I've tried the datasets
- Sample Data from [KHOA S-100 Viewer](https://github.com/S-100ExpertTeam/khoa-s100-viewer/tree/master)
- Test Datasets from [IHO official repo](https://github.com/iho-ohi/S-101-Test-Datasets)
   - .000 files in S-101_Test_DataSets/cells
- Trial Datasets from [UK Admiralty Datahub](https://datahub.admiralty.co.uk/portal/home/item.html?id=6966cb7ce9454ccf9afbbd3c9a105f9e)
These dataset were all cannot be loaded in my circumstance.

I guess that the datasets were not loaded because I built the program quite forcibly.
I could not find VS 2022 distributions so I built program with VS 2026, which would not be required.
I Installed the workload 'Dekstop development with C++' and the following list is the individual components I installed in VS additionally.
- C++ MFC for x64/x86(Latest MSVC)
- MSVC v143 - VS 2022 C++ x64/x86 build tool(+ Spectre-mitigated libs) (v14.44-17.14)
- C++ v14.44(17.14) ATL for v143 build tools(+ with Spectre Mitigations) (x86 & x64)
- C++ v14.44(17.14) MFC for v143 build tools(+ with Spectre Mitigations) (x86 & x64)
Also, I manually installed vcpkg packages and integrated them to VS, but it did not work well, so I specified the include directories for vcpkg packages to VS manually.

P.S.
I arrived this repo while trying to study maritime cybersecurity, especially [CVE-2026-22208](https://www.cve.org/CVERecord?id=CVE-2026-22208). I have some questions about the record. If it is alright, I would like to know the contact details for asking questions.
